rfcs/001-secondary-indexes.md

```diff
@@ -0,0 +1,263 @@
+## Introduction
```

Comment:
Maybe move it inside docs directory?
`docs/rfcs/...`

Reply:
+1 I think moving it inside docs is better.

Reply:
Good idea. I've moved it under the docs.